stories/atoms/components/listItem/ListItem.stories.mdx

```diff
@@ -123,20 +134,211 @@ The `description` property allows you to provide additional information. It can
   </Story>
 </Canvas>
 
-## Right and left elements
+## Grid layout
+
+The style of the listItem can be changed by passing either **list** (default) or **grid** as `layout` value.
```

Comment:
I think this story is not clear enough...
Indeed, the `list` or `grid` value must be passed to the parent `List` element in order to be retrieved by the child. 
Here one might think that it is passed directly to `ListItem`.
Moreover, in the description it says "The style of the listItem can be changed by passing either list (default) or grid", but we only see `ListItem` arranged in grid layout.
Only the content of the `ListItem` changes, which does not fit with the description.

Reply:
This is because it's still possible to change the layout style of an item by passing the property directly. I think it's good to keep the description as is in order to separate `ListItem` from `List`.

For the story I agree I will implement a list layout as well.